src/ukify/ukify.py

```diff
@@ -1571,12 +1570,20 @@ def finalize_options(opts):
         if opts.sb_cert:
             opts.sb_cert = pathlib.Path(opts.sb_cert)
 
-    if opts.signtool == 'sbsign':
-        if bool(opts.sb_key) ^ bool(opts.sb_cert):
-            raise ValueError('--secureboot-private-key= and --secureboot-certificate= must be specified together when using --signtool=sbsign')
-    else:
-        if not bool(opts.sb_cert_name):
-            raise ValueError('--secureboot-certificate-name must be specified when using --signtool=pesign')
+    if bool(opts.sb_key) ^ bool(opts.sb_cert):
+        # one param only given, sbsign need boths
+        raise ValueError('--secureboot-private-key and --secureboot-certificate must be specified together')
```

Comment:
so in systemd so far we said that cmdline options that take parameters should always be suffixed with a "=" in docs and log messages. I think the python code should follow the same rule, i.e. say `--secureboot-private-key=` here, and `--secureboot-certificate=`

Reply:
(here and below)